config/app_config.yml.SAMPLE

```diff
@@ -167,6 +167,14 @@ default: &defaults
   # default to allow automatically adding new articles on sync only when less than 200 articles in total
   #shared_supplier_article_sync_limit: 200
 
+# Multi_coop example configuration
+# In order to make this work set "multi_coop_install: true"
+#anotherfoodcoop:
+#  <<: *defaults
+#  database:
+#    database: foodsoft_anotherfoodcoop
+#  instance_name: anotherfoodcoop
```

Comment:
please move it to the other options like `name`, since it's not more or less special